app/components/rank-step.jsx

```diff
@@ -0,0 +1,67 @@
+//https://github.com/EnCiv/civil-pursuit/issues/65
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import Point from './point.jsx'
+import PointGroup from './point-group.jsx'
+
+import Ranking from './util/ranking.jsx'
+
+function RankStep(props) {
+  const classes = useStylesFromThemeFunction()
+  const {
+    className = '', // may or may not be passed. Should be applied to the outer most tag, after local classNames
+    onDone = undefined, // a function that is called when the button is clicked.  - if it exists
+    children,
+    active = true,
+    pointList = [],
+    rankList = [],
+    ...otherProps
+  } = props
+  const setRank = (index, val) => {
+    for (let i = 0; i < rankList.length; i++) {
+      if (rankList[i].id === index) {
+        rankList[i]['rank'] = val
+        break
+      }
+    }
+  }
+  const findRank = index => {
+    for (let i = 0; i < rankList.length; i++) {
+      if (rankList[i].id === index) {
+        return i
+      }
+    }
+  }
+  return (
+    <div className={cx(classes.wrapper, className)} {...otherProps}>
+      <div className={classes.pointDiv}>
+        {pointList.map((point, i) => (
+          <Point key={point._id} subject={point.subject}>
```

Comment:
please add vState='default' to the Point props, so that the description shows up.
But really the Point component should set vState to default if not passes - but it doesn't.